android/src/main/java/com/bitmovin/player/reactnative/RNPlayerView.kt

```diff
@@ -116,10 +132,28 @@ class RNPlayerView(
         }
 
         override fun onStop(owner: LifecycleOwner) {
+            removePlayerForBackgroundPlayback()
             playerView?.onStop()
         }
 
         override fun onDestroy(owner: LifecycleOwner) = dispose()
+
+        // When background playback is enabled,
+        // remove player from view so it does not get paused when entering background
+        private fun removePlayerForBackgroundPlayback() {
+            playerInMediaSessionService = null
+            playerView?.player?.let {
```

Comment:
Nit: I'm not a big fan of using `let` for nullability checks as the code gets an extra indentation, consider replacing it with:
`val player = playerView?.player ?: return`

It's also generally nice to avoid `it` and name the parameter if the code is more than 1 line.

Reply:
Address in b12d6b09405e5f226371433351402a081b5e60cf